megalinter/MegaLinter.py

```diff
@@ -684,6 +684,9 @@ def initialize_output(self):
             elif os.path.isdir(self.arg_output):
                 # --output /logs/megalinter
                 self.report_folder = self.arg_output
+        # Do not initialize reports if report folder is none or false
+        if self.report_folder == "none" or self.report_folder == "false":
+            return
```

Comment:
For some reason this code has no effect in the latest v6 image on DockerHub.

Setting `REPORT_OUTPUT_FOLDER: none` in the `.mega-linter.yml` config file creates a reports folder called `none` (owned by `root`).

Reply:
Plz can u post a new issue so we don't forget to solve it ? :)

Reply:
Done. #2108